src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java

```diff
@@ -62,8 +51,8 @@
  */
 public class KafkaProtoParquetWriter<T extends Message> implements Closeable {
     private static final Logger logger = LoggerFactory.getLogger(KafkaProtoParquetWriter.class);
+    private final SmartCommitKafkaConsumer<byte[], byte[]> smartCommitKafkaConsumer;
```

Comment:
We used to define constant variables first and then the other ones. In addition, think about the good position of a variable considering the current style you see. 
A good choice for this variable is

Reply:
done

Reply:
done